### DISCUSSION TITLE
System to avoid failures during a match or event.

### DISCUSSION THREAD

**Discussion by Stainded** [diamond-rip/Eden#233] [Ideas]
**Importance Scale(49/100)**

It seems to be something silly but this function can be very important, some clients like Lunar-Client can cause flaws in the gameplay as for example in the "place-block" that makes the movement of defending with the sword and this animation can bug like it happened in other servers such as MinemanClub or you can add a waiting time to add certain commands as I will mention in the example of the image.
![image](https://user-images.githubusercontent.com/128519059/229255214-698477f8-2e1a-4ae6-ab1f-5293e8ddb2f5.png)

**Comment by GoodestEnglish** [comment]
This has been mentioned by someone and I've been considered for this. I will add blocked commands soon.
